package/kernel/linux/modules/netdevices.mk

```diff
@@ -1042,3 +1042,18 @@ define KernelPackage/be2net/description
 endef
 
 $(eval $(call KernelPackage,be2net))
+
+
+define KernelPackage/bnxt
+  TITLE:=Broadcom NetXtreme-C/E driver
+  KCONFIG:=CONFIG_BNXT
+  SUBMENU:=$(NETWORK_DEVICES_MENU)
+  FILES:=$(LINUX_DIR)/drivers/net/ethernet/broadcom/bnxt/bnxt_en.ko
+  AUTOLOAD:=$(call AutoProbe,bnxt)
+endef
+
+define KernelPackage/bnxt/description
+ Kernel modules for Broadcom NetXtreme-C/E Ethernet adapters.
```

Comment:
Nitpick: should be two spaces for indent here.

Reply:
Added 2 spaces.